dashboard-scenario/web-app/src/app/scenario/scenario/list/scenario-list.component.ts

```diff
@@ -0,0 +1,126 @@
+import { Component, OnInit, ViewChild } from '@angular/core';
+import { Scenario } from 'src/app/model/scenario.model';
+import { ScenarioFormComponent } from '../form/scenario-form.component';
+import { ScenarioDialogComponent } from '../dialog/scenario-dialog.component';
+import { ScenarioConfigService } from 'src/app/services/scenario/config/scenario-config.service';
+import { NgForm } from '@angular/forms';
+import { ActivatedRoute, ParamMap } from '@angular/router';
+import { PageSpinner } from 'src/app/core/page-spinner/page-spinner';
+import * as moment from 'moment';
+import { SnackbarControl, SnackbarLevel } from 'objective-design-system';
+import { HttpErrorResponse } from '@angular/common/http';
+
+@Component({
+    selector: 'dbs-scenario-list',
+    templateUrl: './scenario-list.component.html',
+    styleUrls: ['./scenario-list.component.scss']
+})
+
+export class ScenarioListComponent implements OnInit {
+    @ViewChild('addScenario') scenarioForm: ScenarioFormComponent;
+    @ViewChild('scenarioListForm') scenarioListForm: NgForm;
+    @ViewChild('scenarioDialog') scenarioDialog: ScenarioDialogComponent;
+
+    private projectKey: string;
+
+    filterScenarioByFields: string;
+    activeScenario: number;
+    firstActiveScenario: number;
+    scenarios: Scenario[] = [];
+    dialogDescription: string;
+
+    snackbar = new SnackbarControl();
+
+    constructor(
+        private route: ActivatedRoute,
+        private pageLoader: PageSpinner,
+        private scenarioConfigService: ScenarioConfigService
+        ) { 
+    }
+
+    ngOnInit() {
+        this.refreshPage();
+    }
+
+    refreshPage(doneCallback?: Function) {
+        this.pageLoader.show();
+
+        this.route.parent.paramMap.subscribe((params: ParamMap) => {
+            this.projectKey = params.get('key');
+
+            this.scenarioConfigService.getScenarios(this.projectKey).subscribe(scenarios => {
+                this.scenarios = scenarios;
+                this.validateActiveScenario();
+                this.formatScenarioDates();
+                this.pageLoader.hide();
+
+                if (doneCallback !== undefined)
+                    doneCallback();
+            });
+        });
+    }
+
+    validateActiveScenario() {
+        this.activeScenario = this.scenarios.filter(item => item.active)[0].id;
+        this.firstActiveScenario = this.activeScenario;
+    }
+
+    formatScenarioDates() {
+        this.scenarios.forEach(item => {
+            item.formattedDate = moment(new Date(item.creationDate)).format('YYYY-MM-DD HH:mm:ss');
```

Comment:
É meio que padrão nosso na taskboard, o formato é sempre de acordo com o locale do usuário

```suggestion
            item.formattedDate = moment(new Date(item.creationDate)).format('L LTS');
```